.drone.yml

```diff
@@ -94,22 +94,25 @@ pipeline:
       - ENVIRONMENT=${DRONE_DEPLOY_TO}
       - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
     secrets:
-      - hocs_docs_converter_dev
-      - hocs_docs_converter_demo
+      - hocs_docs_converter_cs_dev
+      - hocs_docs_converter_cs_demo
+      - hocs_docs_converter_wcs_dev
```

Comment:
Will the WCS secrets ever get used in this step?

Reply:
I am not sure. We will need to test it in QA.